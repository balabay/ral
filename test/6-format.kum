| Программа к учебнику информатики для 10 класса
| К.Ю. Полякова и Е.А. Еремина.
| Глава 8.
| Программа № 6.
| Вход: нет 
| Результат:
| >  123<
| >  1.235<
алг Форматный вывоод
нач
цел a=123
вывод ">", a:5, "<", нс
вещ x=1.2345678
вывод ">", x:7:3, "<"
кон


