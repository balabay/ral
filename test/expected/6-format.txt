>  123<
>  1.235<